### PR TITLE
CP Scott on Comment is Free container

### PR DESCRIPTION
**Pull request by robertberry-zz**
Mobile:

mobile

Phablet:

mobile-landscape

Tablet:

tablet

Left column:

leftcol

Wide:

wide

### CODE REVIEW

**Comment by robertberry** [comment]
Can I get some love, @sndrs, @phamann, @sammorrisdesign?

File: common/app/layout/Front.scala [hunk 1]
```diff
@@ -254,6 +254,15 @@ case class FaciaContainer(
...
   def hasShowMore = containerLayout.exists(_.hasShowMore)
+
+  /** Nasty hardcoded thing.
+    *
+    * TODO: change Facia Tool to have a dropdown for 'header types', one of which is default, the other CP Scott.
+    *
+    * Then if we end up adding more of these over time, there's an in-built mechanism for doing so. Will also mean apps
+    * can consume this data if they want to.
+    */
+  def showCPScottHeader = dataId == "uk/commentisfree/regular-stories"
```
Comment (sammorrisdesign) [inline common/app/layout/Front.scala]: Do we not want to show this on the US or Australian fronts? Also what happens when the name changes?

File: static/src/stylesheets/module/facia/_cp-scott.scss [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    @include mq(leftCol) {
+        float: none;
+        margin-right: 0;
+    }
+
+    @include mq(wide) {
+        .has-page-skin & {
+            float: right;
+            margin-right: $gs-baseline;
```
Comment (sammorrisdesign) [inline static/src/stylesheets/module/facia/_cp-scott.scss]: Can't use `gs-baseline` for left or right. Use `gs-gutter / 2` to get similar output but still on grid.

**Comment by sammorrisdesign** [comment]
:+1: Other than comments

File: common/app/layout/Front.scala (hunk 1 above)
Comment (theresamalone) [inline common/app/layout/Front.scala]: This is just for the UK opinion front.

File: common/app/views/fragments/containers/facia_cards/cpScottHeader.scala.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@()
+
+@import fragments.inlineSvg
+
+<img src="http://i.guimcode.co.uk.global.prod.fastly.net/static/w-140/h--/q-95/sys-images/Guardian/Pix/pictures/2015/1/22/1421924658711/CPScottcutout.png"
```
Comment (sndrs) [inline common/app/views/fragments/containers/facia_cards/cpScottHeader.scala.html]: this needs to be imagered i think?

File: static/src/stylesheets/module/facia/_cp-scott.scss [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+    .inline-quote svg {
+        height: $gs-gutter / 2;
+        fill: colour(neutral-2);
+    }
+
+    @include mq(mobileLandscape) {
+        clear: none;
+        float: right;
+        margin-right: $gs-baseline;
```
Comment (sndrs) [inline static/src/stylesheets/module/facia/_cp-scott.scss]: as sam says below

**Comment by sndrs** [comment]
be good to confirm with a browserstack on this

File: common/app/layout/Front.scala (hunk 1 above)
Comment (sammorrisdesign) [inline common/app/layout/Front.scala]: But won't the path change to `uk/opinion/regular-stories`

Comment (robertberry) [inline common/app/layout/Front.scala]: Naw. It's just the id of the container, which doesn't change.

**Comment by robertberry** [comment]
Screenshots: http://www.browserstack.com/screenshots/07b03625964d8e36a2166ab512b74eda683aea60

**Comment by sammorrisdesign** [comment]
:+1:

**Pull request merged**